cmd/vpn_server/app/firewall.go

```diff
@@ -90,6 +93,15 @@ func runFirewallCommand(log logr.Logger, device, mode string, networks []string)
 		log.Info(fmt.Sprintf("iptables %s INPUT %s", opName, strings.Join(spec, " ")))
 	}
 
+	err = op4("nat", "PREROUTING", "--in-interface", device, "-d", constants.SeedPodNetworkMapped, "-j", "NETMAP", "--to", seedPodNetwork)
+	if err != nil {
+		return err
+	}
+	err = op4("nat", "POSTROUTING", "--out-interface", device, "-s", seedPodNetwork, "-j", "NETMAP", "--to", constants.SeedPodNetworkMapped)
+	if err != nil {
+		return err
+	}
```

Comment:
Why is this on a completely different path compared to the shoot node, pod, service ranges? Is it because it is device dependent and the device may come and go?